packages/ethers/exports/react.tsx

```diff
@@ -0,0 +1,74 @@
+'use client'
+
+import type { Web3ModalOptions } from '../src/client.js'
+import { Web3Modal } from '../src/client.js'
+import { ConstantsUtil } from '@web3modal/utils'
+import { ProviderController } from '../src/controllers/ProviderController.js'
+import { getWeb3Modal } from '@web3modal/scaffold-react'
+import { useSnapshot } from 'valtio'
+import type { Eip1193Provider } from 'ethers'
+
+// -- Types -------------------------------------------------------------------
+export type { Web3ModalOptions } from '../src/client.js'
+
+// -- Setup -------------------------------------------------------------------
+let modal: Web3Modal | undefined = undefined
+
+export function createWeb3Modal(options: Web3ModalOptions) {
+  if (!modal) {
+    modal = new Web3Modal({
+      ...options,
+      _sdkVersion: `react-ethers-${ConstantsUtil.VERSION}`
+    })
+  }
+  getWeb3Modal(modal)
+
+  return modal
+}
+
+// -- Hooks -------------------------------------------------------------------
+export function useWeb3ModalProvider() {
```

Comment:
Shouldn't we add `signer` to export here, same on how we do it in `ethers-5`? 

This way we keep consistency between ethers packages.